packages/jest-config/src/normalize.js

```diff
@@ -367,6 +372,17 @@ export default function normalize(options: InitialOptions, argv: Argv) {
     ),
   );
 
+  if (options.setupTestFrameworkScriptFile && options.setupFilesAfterEnv) {
```

Comment:
I could push the value of `setupTestFrameworkScriptFile` into the `setupFilesAfterEnv` array here.

Reply:
yes please :)

Should throw if both are defined, but if only `setupTestFrameworkScriptFile` is defined, just push it into `setupFilesAfterEnv` and print a warning